src/arch/x86_64/kernel/irq.rs

```diff
@@ -6,19 +6,21 @@
 // http://opensource.org/licenses/MIT>, at your option. This file may not be
 // copied, modified, or distributed except according to those terms.
 
+pub use x86_64::instructions::interrupts::{disable, enable, enable_and_hlt as enable_and_wait};
```

Comment:
I'm only looking at the diff, but where is that import used?

Reply:
(the `enable_and_hlt` one)

Reply:
`enable_and_hlt` is reexported as `enable_and_wait` and replaces our implementation of the latter. It is used by the scheduler.